sa-eureka-server/src/main/java/com/kyriba/conference/eurekaserver/EurekaServerApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.kyriba.conference.eurekaserver;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer;
+
+
+@EnableEurekaServer
+@SpringBootApplication
+public class EurekaServerApplication

```

Comment:
not needed, we can use docker with Eureka

Reply:
done